priv/repo/migrations/20200609095835_add_affected_release_table.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Hexpm.RepoBase.Migrations.AddPackageReportReleaseTable do
+  use Ecto.Migration
+
+  def up() do
+    execute("""
```

Comment:
please convert the raw sql to use migrations DSL and remember to set null: false on foreign keys.